File: Q3.java
import java.util.Scanner;

public class Q3 
{
    public static Scanner leia = new Scanner(System.in);

    public static void main(String args[]) 
    {
        float numero;
        System.out.print("Digite um número: ");
        numero = leia.nextFloat();
        if (numero >= 0) 
        {
            System.out.println("Esse número é positivo.");
        } 
        else 
        {
            System.out.println("Esse número é negativo.");
        }
    }
}

File: Q3.cpp
#include <iostream>

using namespace std;

int main()
{
    float numero;
    cout << "Digite um número: ";
    cin >> numero;
    if (numero >= 0)
    {
        cout << "Esse número é positivo." << endl;
    }
    else
    {
        cout << "Esse número é negativo." << endl;
    }
    return 0;
}